layouts/community/ortho_4x12/symbolic/config.h

```diff
@@ -20,8 +20,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #pragma once
 
-// hold & tapping delay setting
-#define TAPPING_TERM 100
+// prevent stuck modifiers
+#define PREVENT_STUCK_MODIFIERS
```

Comment:
This setting should be removed, as it's no longer used.

Specifically, it's the new default behavior for QMK Firmware.  And you can disable it by defining `STRICT_LAYER_RELEASE` instead. 

```suggestion
```

Reply:
@drashna , Thank you for your code review. Your work strongly help to keep my codes cleanse and clarify. my codes have been fixed.

Reply:
You're very welcome.  That, and a lot has changed in the last year, so it not only helps to keep the code cleaner, but also lets people know of some of these changes! :)